app/src/jembi/res/drawable/button_primary_background_selector.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
+        <shape android:shape="rectangle">
+            <corners android:radius="24dp" />
+            <solid android:color="@color/fdroid_blue_dark" />
+        </shape>
+    </item>
+    <item android:state_checked="true">
+        <shape android:shape="rectangle">
+            <corners android:radius="24dp" />
+            <solid android:color="@color/fdroid_blue_dark" />
+        </shape>
+    </item>
+    <item android:state_enabled="false">
+        <shape android:shape="rectangle">
+            <corners android:radius="24dp" />
+            <solid android:color="@color/fdroid_blue_dark" />
+        </shape>
+    </item>
+    <item>
+        <shape android:shape="rectangle">
+            <corners android:radius="24dp" />
+            <solid android:color="@color/fdroid_blue" />
+        </shape>
+    </item>
+</selector>
```

Comment:
@patience-mpofu were there any changes in this file? I quickly compared this file with the original one and can't see any changes.

Reply:
The file did not have any changes so I removed it here https://github.com/jembi/fdroidclient/commit/391fd3452ba2a9737dcfeda89cd13d451f123b4e